.travis.yml

```
@@ -40,13 +41,13 @@ jobs:
       env: TOXENV=3.8-linux-unit
     - python: pypy2.7-7.1.1
       env: TOXENV=pypy-unit
-      dist: xenial
     - python: pypy3.5-7.0
       env: TOXENV=pypy3-unit
-      dist: xenial
-    - env: TOXENV=flake8
+    - python: '3.8'
+      env: TOXENV=flake8
       stage: lint
-    - env: TOXENV=flakeplus
+    - python: '3.8'
+      env: TOXENV=flakeplus
```

```suggestion
    - python: '2.7'
      env: TOXENV=flakeplus
```

This one should be for 2.7 as it tests 2.7 related checks.
We dropped it in Celery 5.

---

See build for the obvious failure:
https://travis-ci.org/github/celery/kombu/jobs/701585171